imagetest/test_suites/metadata/shutdown_script_test.go

```diff
@@ -45,7 +46,7 @@ func TestShutdownUrlScript(t *testing.T) {
 
 // TestShutdownScriptTime test that shutdown scripts can run for around two minutes
 func TestShutdownScriptTime(t *testing.T) {
-	bytes, err := ioutil.ReadFile("/shutdown.txt")
```

Comment:
this path is set in setup.go, so if you want to change it here, you need to change it there as well.

Reply:
I didn't want to change it. I thought this test forgot to use the contant in setup.go. Just read it again, realized it's different (shutdown_out.txt vs shutdown.txt")